packages/web-app-files/src/spaces/sdk/index.ts

```diff
@@ -0,0 +1,30 @@
+import axios, { AxiosPromise } from 'axios'
+import { CollectionOfDrives, Configuration, MeDrivesApi } from './generated'
+
+export const spacesSDK = (baseURI: string, token: string): any => {
+  const basePath = new URL('/graph/v1.0', baseURI).href
+
+  const config = new Configuration({
+    basePath
+  })
+
+  const axiosClient = axios.create({
+    headers: {
+      authorization: 'Bearer ' + token,
+      'Content-Type': 'application/x-www-form-urlencoded'
+    }
+  })
+
+  axiosClient.interceptors.request.use((config) => {
+    config.headers['Content-Type'] = 'application/x-www-form-urlencoded'
+    return config
+  })
+
+  const meDrivesApi = new MeDrivesApi(config, config.basePath, axiosClient)
+
+  return {
+    drives: {
+      listMyDrives: (): AxiosPromise<CollectionOfDrives> => meDrivesApi.listMyDrives()
```

Comment:
can be `listMyDrives: meDrivesApi.listMyDrives`

Reply:
This is not working for me, spaces list will be empty then :(

Reply:
i can confirm, it errors due the BASE_PATH. Lets keep it as it is